zip-0316.rst

```diff
@@ -416,6 +440,21 @@ Requirements for both Unified Addresses and Unified Viewing Keys
   that cannot be interpreted as specified above.
 
 
+Adding new types
+----------------
+
+It is intended that new Receiver Types and Viewing Key Types SHOULD be
+introduced either by a modification to this ZIP or by a new ZIP, in accordance
+with the ZIP Process [#zip-0000]_.
+
+Experimental types MAY be added using the reserved Typecodes 0xFFFA to 0xFFFF
```

Comment:
It's unclear whether this requires a ZIP, given the wording above.

Suggestion: move most of this section up to where the typecodes are defined, and say here "for experimentation prior to proposing a ZIP".